projects/ng7-mat-breadcrumb/src/lib/ng7-mat-breadcrumb.component.ts

```diff
@@ -70,13 +83,15 @@ export class Ng7MatBreadcrumbComponent implements OnInit {
               for (const labelParam of labelParams) {
                 const routerParamID = route.snapshot.params[labelParam.trim()];
                 if (routerParamID) {
-                  crumb.label = crumb.label.replace('{{' + labelParam + '}}', routerParamID);
+                  crumb.displayText = crumb.label.replace(
+                    '{{' + labelParam + '}}',
+                    routerParamID
+                  );
```

Comment:
Instead of overwriting the label, set the displayText and leave the label intact.